test/Microsoft.Azure.WebJobs.Host.EndToEndTests/ParallelExecutionTests.cs

```diff
@@ -88,25 +85,32 @@ public async Task MaxDegreeOfParallelism_Queues(int batchSize, int maxExpectedPa
             IHost host = new HostBuilder()
                 .ConfigureDefaultTestHost<ParallelExecutionTests>(b =>
                 {
-                    b.AddAzureStorage();
+                    b.AddAzureStorageQueues(o =>
```

Comment:
Track2 Extensions don't do case-insensitive comparison for consumption ("dynamic") environment varaible but Track1 does. 
Track1: https://github.com/Azure/azure-webjobs-sdk/blob/da78eb482e5b2d44fae6b911bc11bddefff7abd7/src/Microsoft.Azure.WebJobs.Host.Storage/Utility.cs#L28
Track2: https://github.com/Azure/azure-sdk-for-net/blob/2103c3801199eeaa0cd8b762b28658f8223ffb02/sdk/storage/Microsoft.Azure.WebJobs.Extensions.Storage.Common/src/Shared/SkuUtility.cs?plain=1#L13